docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  fxa-oauth-console:
+    build:
+      context: .
+      dockerfile: Dockerfile-build
+    command: node bin/server.js
+    image: mozilla/fxa-oauth-console
+    ports:
+      - "10137:10137"
```

Comment:
When running the container with other servers on the HOST I get the following: 

```
fxa-oauth-console.CRITICAL: uncaughtException TypeError: Cannot read property 'status' of undefined
    at Request._callback (/app/lib/routes/oauth.js:102:26)
    at self.callback (/app/node_modules/request/request.js:186:22)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
    at Request.onRequestError (/app/node_modules/request/request.js:845:8)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at Socket.socketErrorListener (_http_client.js:269:9)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
```

I think this is because the OAuth server is outside of the container. It cannot make this request: https://github.com/mozilla/fxa-oauth-console/blob/master/lib/routes/lib/oauth.js#L51

Using a different Docker networking mode might help with this. Need to investigate...